internal/assets/manager.go

```diff
@@ -59,6 +67,12 @@ func NewAssetManager(ctx context.Context, di database.Plugin, ii identity.Plugin
 		data:      dm,
 		syncasync: sa,
 		tokens:    ti,
+		retry: retry.Retry{
+			InitialDelay: config.GetDuration(config.EventAggregatorRetryInitDelay),
+			MaximumDelay: config.GetDuration(config.EventAggregatorRetryMaxDelay),
```

Comment:
Think this should be a dedicated retry config under `assets.manager`

Reply:
Yea, kinda thought that as I was copy-pasting it 😇 I'll add a new config section